libs/ui/src/lib/modules-graph/modules-graph.component.ts

```diff
@@ -0,0 +1,245 @@
+import { Component, ElementRef, Input, ViewChild } from '@angular/core';
+import { select, selectAll } from 'd3-selection';
+import { arc } from 'd3-shape';
+import { hierarchy, partition } from 'd3-hierarchy';
+import 'd3-transition';
+import { FormatFileSizePipe } from '../format-file-size.pipe';
+
+interface Chunk {
```

Comment:
Consider running much of the logic within this component outside of the angular zone. D3 is known for taking a huge performance hit when running inside of a zone